app/services/geoapify_service.rb

```diff
@@ -0,0 +1,12 @@
+class GeoapifyService 
+  def self.conn 
+    Faraday.new("https://api.geoapify.com")
+  end 
+end 
+
+def self.tourist_sight(capital)
```

Comment:
Youre sending this a country, which is a string but this is expecting a captial object. What we were looking for was for you to send the country from params to the countryfacade object, get a capital object and then send that capital object here so you can get the lat and long to get the sights.